pytest.ini

```diff
@@ -3,7 +3,7 @@ log_cli = 1
 log_cli_level = ERROR
 log_cli_format = %(message)s
 log_file = pytest.log
-log_file_level = DEBUG
+log_file_level = INFO
```

Comment:
this should be kept DEBUG because pytest is only used for development. 

When there are tests failing or the code has problems you usually want to show in pytest.log 
and add debug messages.